tests/processing/test_process_queue.py

```
@@ -0,0 +1,42 @@
+from unittest.mock import patch, Mock
+from io import BytesIO
+from app.processing.process_queue import extractive_summary_process
+from app.processing.process_queue import highlight_sentences_in_pdf
+from app.processing.process_queue import extract_summary
+import os
+import builtins
+
+@patch('app.processing.process_queue.extract_summary')
+@patch('app.processing.process_queue.highlight_sentences_in_pdf')
+@patch('os.path.isfile')
+@patch('builtins.open', new_callable=mock_open, read_data="test file content")
```

Ruff

Undefined name `mock_open`

(F821)